### PR TITLE
re-former using rails

### PR DESCRIPTION
**Pull request by karthykarthick**
in this rails tutorial, we implemented forms project

### CODE REVIEW

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello @karthykarthick 
### You have done well so far :clap: However, you have a few issues to fix:-
- [ ] Finish all the required actions in the controller ie `new, create, edit, update`
- [ ] Update the routes to be more descriptive to the actions ie `resources :users, only: [:new, :create, :edit, :update]`
- [ ] commented out `form`
```html
<form action="users_path" method="post" accept-charset="UTF-8">
  <label for="user[username]">Username</label>
  <input name="user[username]" type="text" />
  <input
    type="hidden"
    name="authenticity_token"
    value="<%= form_authenticity_token %>"
  />
  <input type="submit" value="Submit" />
</form>
```
- [ ] commented out `form_tag`
```html
<%= form_tag(users_path) do %> <%= label_tag 'username' %> <%= text_field_tag
'username' %> <%= label_tag 'email' %> <%= email_field_tag 'email' %> <%=
label_tag 'password' %> <%= password_field_tag 'password' %> <%= submit_tag %>
<% end %>
```
- [ ] Uses `form_for`
```html
<%= form_for @user do |u|%> <%= u.label :username %> <%= u.text_field :username
%> <%= u.label :email %> <%= u.email_field :email %> <%= u.label :password %>
<%= u.password_field :password %> <%= u.submit %> <% end %>
```
- [ ] Add github actions, rubocop and stylelint in the project. [Follow this guide](https://github.com/microverseinc/linters-config/tree/master/ror)
- [ ] Write a descriptive readme file and pull request
### Make the changes and submit for another code review. Good luck.
- Happy coding!

File: app/views/users/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  <% @users.each do |user| %>
+    <li>
+        <%= 'Username = ' + user.username  %><br>
+        <%= 'Email = ' + user.email  %><br>     
+          <%= link_to "Edit User", edit_user_path(user.id) %>
+          <br> <br> 
+    </li>
+    <% end %>
+</ul>
+<% end %>
```
Comment (lekegitrepo) [inline app/views/users/new.html.erb:34]: - **_As mentioned in the previous review: There should be an Html form which should be commented out_**

- **_As mentioned in the previous review: There should be a form_tag which should be commented out_**

**Review by lekegitrepo — CHANGES_REQUESTED** [review]
## Status: Request changes :recycle:
#### Good job so far :+1: However, you need to implement the remaining requirements mentioned in the previous review.

- **_Add a descriptive Pull request message at the top where the placeholder "No description provided" is._**

#### _Reviewed by Temitayo Adeleke_

**Comment by karthykarthick** [comment]
in this rails tutorial, we implemented forms project

**Pull request closed without merging**